test/logs/color/color.go

```diff
@@ -1,5 +1,6 @@
 /*
-Copyright 2017 Google Inc. All Rights Reserved.
+Copyright 2019 The Tekton Authors
```

Comment:
Perhaps this code was copied in from somewhere else?
I think it's fine to update the copyright statement. If the code was copied it would be good to have a comment pointing to the original source, but who knows?